src/swaps/blindex/BlindexSwapProvider.js

```diff
@@ -239,7 +238,7 @@ class BlindexSwapProvider extends SwapProvider {
     const blockHeight = await client.chain.getBlockHeight()
     const currentBlock = await client.chain.getBlockByNumber(blockHeight)
 
-    const path = [fromToken.address, toToken.address]
+    const path = this.bestRoute
```

Comment:
you can just use `this.bestRoute` instead of creating another variable

Reply:
done